api/pkg/apis/v1alpha1/model/summary.go

```diff
@@ -20,7 +21,6 @@ type ComponentResultSpec struct {
 }
 type TargetResultSpec struct {
 	Status           string                         `json:"status"`
-	Message          string                         `json:"message,omitempty"`
```

Comment:
@Haishi2016 , we are going to remove the target.message which is duplicated with component result. Besides the changes in symphony-api, will this affect the OSS portal?